mne/decoding/transformer.py

```diff
@@ -31,23 +39,41 @@ class Scaler(TransformerMixin):
 
     Attributes
     ----------
+    scaler : dict, string, defaults to "standard"
+        if scaler == 'robust' sklearn.proprocessing.RobustScaler() is used.
+        if scaler == 'standard' applies ``X/std-mean``
+        if scaler == 'default', scales mag by 1e15
+                                       grad by 1e13
+                                       eeg by 1e6
     info : instance of Info
         The measurement info
     ``ch_mean_`` : dict
         The mean value for each channel type
     ``std_`` : dict
         The standard deviation for each channel type
-    """
-
-    def __init__(self, info, with_mean=True, with_std=True):  # noqa: D102
+     coef_ : array, shape (n_channels,)
+        The scaling coefficients.
+     """
+ 
+    def __init__(self, scalings='standard', info=None, with_mean=True,
+                 with_std=True):
+        from sklearn.preprocessing import RobustScaler
         self.info = info
         self.with_mean = with_mean
         self.with_std = with_std
-        self.ch_mean_ = dict()  # TODO rename attribute
-        self.std_ = dict()  # TODO rename attribute
+
+        if (isinstance(scalings, str) and scalings not in
+                      ('robust', 'standard', 'default')):
+            raise ValueError("Invalid method for scaling")
+
+        if scalings == 'robust':
+            scalings = RobustScaler()
```

Comment:
pass with_mean, with_std in there? @agramfort is this ok to set a param like this init? I guess it will not clone properly?